.yarnrc

```diff
@@ -1 +1,6 @@
-unsafe-disable-integrity-migration false
\ No newline at end of file
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+unsafe-disable-integrity-migration false
+yarn-path ".yarn/releases/yarn-1.19.0.cjs"
```

Comment:
For some reason i could no longer yarn add etc, due to https://github.com/yarnpkg/yarn/issues/7807. This fixed it